File: Problem_set_6/cash/cash.py
dol = -1
coins= 0
cents = 1
while (dol < 0):
    try:
        dol = float(input("Change owed: "))
    except ValueError:
        dol = -1
cents = round(m * 100)
if (cents >= 25):
    while (cents >= 25):
        cents -= 25
        coins+= 1
if (cents >= 10):
    while (cents >= 10):
        cents -= 10
        coins+= 1
if (cents >= 5):
    while (cents >= 5):
        cents -= 5
        coins+= 1
if (cents < 5):
    while (cents > 0):
        cents -= 1
        coins+= 1
print("{}".format(coins))

File: Problem_set_1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    float m = -1;
    int i = 0, n = 1;

    while (m < 0)
    {
        m = get_float("Change owed: ");
    }
    n = round(m * 100);
    if (n >= 25)
    {
        while (n >= 25)
        {
            n -= 25;
            i++;
        }
    }
    if (n >= 10)
    {
        while (n >= 10)
        {
            n -= 10;
            i++;
        }
    }
    if (n >= 5)
    {
        while (n >= 5)
        {
            n -= 5;
            i++;
        }
    }
    if (n < 5)
    {
        while (n > 0)
        {
            n -= 1;
            i++;
        }
    }
    printf("%i\n", i);
}
